data/json/mapgen/animalshelter.json

```diff
@@ -139,6 +139,39 @@
       { "monster": "mon_dog_samoyed_pup", "weight": 5, "cost_multiplier": 0 }
     ]
   },
+  {
+    "name": "GROUP_ANIMALSHELTER_DOGS_NOZOM",
```

Comment:
Nitpick, but we already have `GROUP_PET_DOGS` monster group, which has exactly the same monsters and weights as in this one.